Remove `display:block` on hint component (#2437)
alphagov/govuk-frontend: CHANGELOG.md

This change was introduced in [pull request #1836: Rename exported JavaScript modules to include component name](https://github.com/alphagov/govuk-frontend/issues/1836)].

#### Update the HTML for hints

> In GOV.UK Frontend v3.8.0, we [updated the HTML for hints to use a `<div>` rather than a `<span>` element, to allow the use of block elements](https://github.com/alphagov/govuk-frontend/issues/1835). We've now removed the styles which made the hint <span> display as block.

Reviewer: ```suggestion
In GOV.UK Frontend v3.8.0, we [updated the HTML for hints to use a `<div>` rather than a `<span>` element, to allow the use of block elements](https://github.com/alphagov/govuk-frontend/issues/1835). We've now removed the styles which made the hint `<span>` display as block.
```

This change was introduced in [pull request #1836: Rename exported JavaScript modules to include component name](https://github.com/alphagov/govuk-frontend/issues/1836)].

#### Update the HTML for hints

> In v3.8.0, we updated the HTML for hints to use a <div> rather than a <span> element, to allow block elements to be used.

Reviewer: ```suggestion
In [GOV.UK Frontend v3.8.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.8.0), we updated the HTML for hints to use a `<div>` rather than a `<span>` element, to allow the use of block elements.
```
In the v3.8.0 release notes, the link for this change is in the 'Fixes' section, which might make it easy to miss. Wonder is there a case for linking to [#1855](https://github.com/alphagov/govuk-frontend/pull/1855) from the text "updated the HTML for hints...use of block elements"?

> We have now removed the styles which made the hint <span> display as block.

Reviewer: ```suggestion
We've now removed the styles which made the hint `<span>` display as block.
```

> Update your hint HTML to use a <div> rather than a <span>, for example: `<div id="input-hint" class="govuk-hint"></div>`, to ensure your hints are styled correctly.

Reviewer: ```suggestion
To make sure your hints are styled correctly, update your hint HTML to use a `<div>` rather than a `<span>`. For example: `<div id="input-hint" class="govuk-hint"></div>`.
```
